### ISSUE TITLE
Add a loading animation to the change state view when sending emails 

### ISSUE THREAD

**Issue by Gravewalker666** [sef-global/scholarx-frontend-v1#116]
**Is your feature request related to a problem? Please describe.**
When changing the states some emails notifications are sent to the related users and it takes a bit of time to send those emails. So It'll be nice to have some sort of a notification and a loading animation whilst emails are sent.

**Describe the solution you'd like**
Trigger a loading animation when the change button is clicked and switch it off once the task is done.

**Comment by Gravewalker666** [comment]
Another issue is created for the issue (#117)
